docusaurus/docs/tools/user_guide/poktrolld_cli.md

```diff
@@ -76,38 +70,58 @@ brew tap pokt-network/poktroll
 brew install poktrolld
 ```
 
-### From Source
-
-#### Installing dependencies
+</details>
 
-Ensure you have the following installed:
+### Using release binaries
 
-- [Go](https://go.dev/doc/install) (version 1.18 or later)
-- [Ignite CLI](https://docs.ignite.com/welcome/install)
+Pre-built binaries are available on our [releases page](https://github.com/pokt-network/poktroll/releases).
 
-If you're on a Linux machine, you can follow the steps below for convenience:
+The following snippet downloads/upgrades the binary to the latest released version:
 
 ```bash
-# Install go 1.23
-curl -o ./pkgx --compressed -f --proto '=https' https://pkgx.sh/$(uname)/$(uname -m)
-sudo install -m 755 pkgx /usr/local/bin
-pkgx install go@1.23.0
-export PATH=$PATH:$HOME/go/bin/
+# Download the correct binary based on the OS and architecture
+curl -LO "https://github.com/pokt-network/poktroll/releases/latest/download/poktroll_$(uname | tr '[:upper:]' '[:lower:]')_$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/').tar.gz"
 
-# Install PATH Gateway required dependencies
-apt-get update && apt-get install git make build-essential
+# Extract the downloaded tarball to /usr/local/bin
+sudo tar -zxf "poktroll_$(uname | tr '[:upper:]' '[:lower:]')_$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/').tar.gz" -C /usr/local/bin
 
-# Install the ignite binary used to build the Pocket binary
-curl https://get.ignite.com/cli! | bash
+# Make the binary executable
+sudo chmod +x /usr/local/bin/poktrolld
+
+# Check version
+poktrolld version
 ```
 
+### From Source
+
+:::warning
+
+This method is only recommended for advanced users as it requires working with developer tools.
```

Comment:
I removed the linux snippet because:
- from one system/package manager to another, commands will differ
- we expect people who use this installation path to understand what they are doing. 
- keep it light and simple.